Make sure the build system honors CPATH and LIBRARY_PATH when looking for
headers and libraries.

--- setup.py	2015-10-07 23:32:58.891329173 +0200
+++ setup.py	2015-10-07 23:46:29.653349924 +0200
@@ -507,11 +507,8 @@ class PyBuildExt(build_ext):
         # if a file is found in one of those directories, it can
         # be assumed that no additional -I,-L directives are needed.
         if not cross_compiling:
-            lib_dirs = self.compiler.library_dirs + [
-                '/lib64', '/usr/lib64',
-                '/lib', '/usr/lib',
-                ]
-            inc_dirs = self.compiler.include_dirs + ['/usr/include']
+            lib_dirs = os.getenv('LIBRARY_PATH', '').split(os.pathsep)
+            inc_dirs = os.getenv('CPATH', '').split(os.pathsep)
         else:
             lib_dirs = self.compiler.library_dirs[:]
             inc_dirs = self.compiler.include_dirs[:]
